cpp/include/cuml/decomposition/pca.hpp

```diff
@@ -24,7 +24,7 @@ namespace ML {
 void pcaFit(cumlHandle &handle, float *input, float *components,
             float *explained_var, float *explained_var_ratio,
             float *singular_vals, float *mu, float *noise_vars,
-            const paramsPCA prms);
+            const paramsPCA &prms);
```

Comment:
Realising that there was a missing `&` here took me a good long time. gtests don't capture this as it is merely syntax to pass by reference without needing a pointer. Cython, on the other hand, gives the very valuable and descriptive error, `Undefined Symbol *mostly gibberish*`